maven-plugin/src/it/default-configuration/postbuild.groovy

```diff
@@ -0,0 +1,12 @@
+log = new File(basedir, 'build.log');
+sorted = new File(basedir, 'pom.xml');
+expected = new File(basedir, 'src/test/resources/pom.xml');
+backup = new File(basedir, 'pom.xml.bak');
+
+assert log.exists();
+assert log.text.contains('Sorting file ' + sorted.absolutePath);
+assert log.text.contains('Pom file is already sorted, exiting');
+assert !backup.exists();
+assert expected.text.replaceAll('@pom.version@', projectversion).tokenize('\n').equals(sorted.text.tokenize('\n'));
+
+return true;
```

Comment:
The postbuild script, in groovy

It checks that the plugin behaves as expected, by checking the log of the build, and also the files produced (or not produced) by the plugin

The _expected.text.replaceAll('@pom.version@', projectversion)_ is here to replace the _@pom.version@_ of the _src/test/resources/pom.xml_ with the current version of the plugin, since in the _pom.xml_ it has already been done by the invoker plugin